cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -128,10 +147,14 @@ def download_data(dataset: str) -> FlyteDirectory:
         input=download_subprocess.stdout,
     )
 
-    # return the directory populated with rossmann data files
+    # return the directory populated with Rossmann data files
     return FlyteDirectory(path=str(data_dir))
 
-
+# %%
+# Data Preprocessing
+# =====================
+#
+# Here, we clean and prepare the Google trend data.
```

Comment:
```suggestion
# We clean and prepare the Google trend data.
```

Reply:
Can we have 1., 2., ... for every data preprocessing step?

Reply:
Smriti, let's either explain every data preprocessing function clearly or just say that we are doing Spark data preprocessing. If we explain what each function does without talking about how it actually contributes to data preprocessing, the semantics is really confusing and unfinished. Hope you got my point.